corehq/apps/hqcase/utils.py

```diff
@@ -35,7 +35,7 @@
 
 def submit_case_blocks(case_blocks, domain, username="system", user_id=None,
                        xmlns=None, attachments=None, form_id=None,
-                       form_extras=None, case_db=None, device_id=None):
+                       form_extras=None, case_db=None, device_id=None, max_wait=Ellipsis):
```

Comment:
Is `Ellipsis` an alternate constant because `None` will not work here? Probably would be more standard to use the ellipsis literal instead
```suggestion
                       form_extras=None, case_db=None, device_id=None, max_wait=...):
```